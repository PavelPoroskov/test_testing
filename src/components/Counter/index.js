import React from 'react'
import PropTypes from 'prop-types'

export default
function Counter({counter}) {
  return (
    <p data-testid='counter'>{counter}</p>
  )
}

Counter.propTypes = {
  counter: PropTypes.number
}
