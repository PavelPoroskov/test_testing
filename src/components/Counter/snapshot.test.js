import renderer from 'react-test-renderer';
import React from 'react';
import Counter from './index'

test('shanshot Counter', () => {
  const tree = renderer
    .create(<Counter counter={0} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});