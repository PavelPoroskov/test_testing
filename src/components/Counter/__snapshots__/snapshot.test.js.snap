// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`shanshot Counter 1`] = `
<p
  data-testid="counter"
>
  0
</p>
`;
